docs/configuration/configuration.md

```diff
@@ -893,6 +893,7 @@ The following meta labels are available on targets during [relabeling](#relabel_
 * `__meta_ec2_ami`: the EC2 Amazon Machine Image
 * `__meta_ec2_architecture`: the architecture of the instance
 * `__meta_ec2_availability_zone`: the availability zone in which the instance is running
+* `__meta_ec2_availability_zone_id`: the underlying availability zone in which the instance is running
```

Comment:
Maybe we should just do one last update and better explain the difference between the two labels?

Reply:
How about `the [underlying](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html) availability zone of the instance, that is consistent between accounts`?

Reply:
Why not simply `the [availability zone ID](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html) in which the instance is running` as this is what I read in the docs?